tests/checks/dangling-servicemonitor.yml

```diff
@@ -0,0 +1,77 @@
+apiVersion: v1
```

Comment:
No test for for ServiceMonitors with both `selector` and `namespaceSelector`?

Reply:
added a couple in bc5b23be63144ac60428255f8646a058d37acdea to test:
(1) when labelSelector and namespaceSelector both match a service
(2) when labelSelector matches a service, but namespaceSelector doesn't